### PR TITLE
Publish solutions for lab 11

### PR DESCRIPTION
**Pull request by andreia-oca**

### CODE REVIEW

File: labs/lab11/skel/lin/1-pollpipe/poll.c [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
 #include <wait.h>
 #include <string.h>
 
 #include "utils.h"
 
 #define CLIENT_COUNT	10
-
 #define MSG_SIZE	32
 
 #define	ANY		-1
```
Comment (sketch873) [inline labs/lab11/skel/lin/1-pollpipe/poll.c]: Add parantheses

File: labs/lab11/skel/lin/2-epollpipe/epoll.c [hunk 2]
```diff
@@ -10,14 +10,12 @@
...
 #include <sys/epoll.h>
 #include <wait.h>
 #include <string.h>
 #include <sys/eventfd.h>
 
 #define CLIENT_COUNT	10
-
 #define MSG_SIZE	32
 
 #define	ANY		-1
```
Comment (sketch873) [inline labs/lab11/skel/lin/2-epollpipe/epoll.c]: Add parantheses

File: labs/lab11/sol/lin/1-pollpipe/poll.c [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+#include <poll.h>
+#include <wait.h>
+#include <string.h>
+
+#include "utils.h"
+
+#define CLIENT_COUNT	10
+#define MSG_SIZE	32
+
+#define	ANY		-1
```
Comment (sketch873) [inline labs/lab11/sol/lin/1-pollpipe/poll.c]: Add parantheses

File: labs/lab11/sol/lin/2-epollpipe/epoll.c [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+#include <unistd.h>
+#include <sys/epoll.h>
+#include <wait.h>
+#include <string.h>
+#include <sys/eventfd.h>
+
+#define CLIENT_COUNT	10
+#define MSG_SIZE	32
+
+#define	ANY		-1
```
Comment (sketch873) [inline labs/lab11/sol/lin/2-epollpipe/epoll.c]: Add parantheses

**Review by sketch873 — DISMISSED** [review]
Everything looks fine. The only change I would make is at some defines, some parantheses.

**Review by sketch873 — APPROVED** [review]

**Review by lucigrigo — APPROVED** [review]
Looks good.

**Pull request merged**